src/epgsql.erl

```diff
@@ -243,3 +243,12 @@ sync_on_error(C, Error = {error, _}) ->
 sync_on_error(_C, R) ->
     R.
 
+send_or_terminate(Connection, Request) ->
+    gen_server:call(Connection, Request, infinity).
+    %case gen_server:call(Connection, Request, ?EXEC_MAX_WAIT) of
```

Comment:
It's the latter.  I had orginally done changes to that module and found that it worked - but I ended up touching a lot of things because of how many incremental calls get pipelined. I have most of that stashed locally, the changes in this file were the bits I pushed up before that point. 

The current `epgsql_sock` implementation replaces these changes (both those pushed and those stashed ).